ci-operator/config/terraform-redhat/terraform-rhcs-rosa/terraform-redhat-terraform-rhcs-rosa-main.yaml

```
@@ -0,0 +1,28 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- dockerfile_path: Dockerfile
+  to: clis-img
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: verify-format
+  commands: make init && make verify
```

```suggestion
  commands: make init verify
```
or even better, make ``verify`` depend on ``init`` target

---

Do you mean changing it at the component Makefile?

---

yes,
```makefile
init:
    ...
verify: init
   ...
```